cms/src/main/java/uk/nhs/digital/ps/migration/ResolveLinksHelper.java

```diff
@@ -0,0 +1,39 @@
+package uk.nhs.digital.ps.migration;
```

Comment:
this package will be deleted shortly after GOSS migration. If this helper should exist post 19th April, please move it to another package.